src/components/Chart/Display.svelte

```diff
@@ -0,0 +1,120 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+    import { flow } from './mock.ts';
+    import Chart from 'chart.js/auto';
+    import type { ChartConfiguration } from 'chart.js';
+    import 'chartjs-adapter-date-fns';
+    import ChartStreaming from 'chartjs-plugin-streaming';
+    import ChartZoom from 'chartjs-plugin-zoom';
+    import colors from 'tailwindcss/colors';
```

Comment:
Just reordering some imports here. I basically alphabetized by the name of the third-party package.
```suggestion
    import type { ChartConfiguration } from 'chart.js';
    import Chart from 'chart.js/auto';
    import ChartStreaming from 'chartjs-plugin-streaming';
    import ChartZoom from 'chartjs-plugin-zoom';
    import { onMount } from 'svelte';
    import colors from 'tailwindcss/colors';

    import 'chartjs-adapter-date-fns';

    import { flow } from './mock.ts';
```